src/org/mozilla/javascript/AbstractEcmaObjectOperations.java

```diff
@@ -24,6 +24,32 @@
         SEALED
     }
 
+    /**
+     * Implementation of Abstract Object operation HasOwnProperty as defined by EcmaScript
+     *
+     * @param cx
+     * @param o
+     * @param pn
```

Comment:
I'd prefer a more descriptive name, for example 'property'

Reply:
Thank you for pointing out! I refactor at https://github.com/mozilla/rhino/pull/1157/commits/de372ec5f245dbc205351d11212032d5b7f30ee6